uint32 height
uint32 width
uint8 fps
bool is_toconf
bool is_running
---
bool success
